cleangoldenami/regions.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+var regions = []string{
```

Comment:
Can we have this whole list/file be replaced by an aws API call where we get all the regions?

Reply:
Great suggestion - done in latest commit (+ tests)